tests/base/develop_pipelines.py

```diff
@@ -56,7 +61,8 @@ def run_model_test(trainer_options, model, on_gpu: bool = True, version=None, wi
     logger = get_default_logger(save_dir, version=version)
     trainer_options.update(logger=logger)
 
-    if 'checkpoint_callback' not in trainer_options:
+    # TODO: DEPRECATED option
+    if "checkpoint_callback" not in trainer_options:
         trainer_options.update(checkpoint_callback=True)
```

Comment:
shall we remove it?

Reply:
I'll fix this in order to make it right as part of this PR :)

Reply:
Hi @Borda, I think we can remove it as it's not working anymore. We can add this feature in the feature :)